app/Router.tsx

```diff
@@ -61,16 +58,13 @@ export const Router = () => {
             </SecondaryNavigationWrapper>
             <ServiceListingPage />
           </>
-        )}
+        }
       />
-      <Route exact path="/terms-of-service" component={TermsOfServicePage} />
+      <Route path="/terms-of-service" element={<TermsOfServicePage />} />
       <Route
-        exact
         path="/:categorySlug/results"
-        component={ServiceDiscoveryResults}
+        element={<ServiceDiscoveryResults />}
       />
-
-      <Redirect to="/" />
```

Comment:
**note:** This is a vestige of how `Switch` was used in v5. Although looking at it I'm not sure how it was supposed to operate since there is no `from` prop declared. 

See https://gist.github.com/mjackson/b5748add2795ce7448a366ae8f8ae3bb for more explanation from React Router team about how to handle redirects on initial load. TLDR: do it on the server.